docs/node/migrations/offline-migrations.mdx

```diff
@@ -0,0 +1,142 @@
+
+```json
+{
+  "genesis_time": "2024-04-15T17:00:34.108598516Z",
+  "chain_id": "kwil-chain-g1KPHOsY",
+  "activations": {
+    "halt": 1234000 // `halt` set to agreed upon block height
+  },
+  ...
+}
+```
+Stop the node once the `halt` activation height is reached.
+
+## Create Network Snapshot
+
+Use the `kwil-admin snapshot` tool to take a snapshot of the final state of the Kwild database. For more details, refer to the [create database snapshots](/docs/ref/kwil-admin/snapshot/create) documentation. This tool connects directly to the database to capture its state and does not require the `kwild` process to be running.
+
+```sh
+kwil-admin snapshot create --snapdir /path/to/snapdir
+```
+
+:::tip
+By default, the `snapshot create` command connects to the `postgres` instance running at `localhost:5432` with the default user `postgres` (without a password?). To modify the database connection settings, see the `kwil-admin` reference page.
+:::
+
+This generates a compressed snapshot file and a template genesis file required to start the new network as shown below.
+
+```sh
+$ ls /path/to/snapdir
+genesis.json            kwildb-snapshot.sql.gz
+```
+
+### Verify App Hash
+
+The app hash set in the genesis file is generated based on the hash of the uncompressed snapshot. You can verify the app hash in the generated genesis file as follows
+
+``` bash
+kwil-admin setup genesis-hash --snapshot /path/to/snapdir/kwildb-snapshot.sql.gz
+```
+
+## Distribute Genesis State
+
+The generated genesis file includes the genesis app hash, derived from the hash of the uncompressed snapshot file, and the genesis validators set to the validators from the previous network. If there are any changes to the validator set in the new network, modify the validator set manually to remove or add new validators. The rest of the configuration in the genesis file is set to defaults, so you should update it based on your requirements before distributing the genesis file.
+
+:::warning
+It is recommended that only one validator operator is responsible for generating, updating, and distributing the genesis state file to all other validators. This ensures that all validators are working from the same, consistent configuration.
+:::
+
+Distribute the `kwildb-snapshot.sql.gz` and the `genesis.json` files required to start the new network. 
+
+## Start New Network
+
+### Install Kwild Binaries
+
+Install the new version of the `kwild` binaries. Refer to the [Installation](/docs/daemon/installation) documentation for more details on how to install the `kwild` binaries.
+
+### Setup Root directory
+
+Copy the `kwildb-snapshot.sql.gz`, `genesis.json` files downloaded [above](/docs/node/migrations/offline-migrations#create-network-snapshot) to the new node's root directory. 
```

Comment:
Instead of having them copy it, why don't you do:

```
kwil-admin setup init --root-dir /new/node/dir --genesis ./path/to/genesis.json --genesis-state ./path/to/snapshot.sql.gz
```

This will remove the need for them to mess with their node data dir, as well as remove the step where they need to set their `app.genesis_state` configuration